packages/web3torrent/src/components/torrent-info/channels-list/ChannelsList.tsx

```diff
@@ -67,12 +67,14 @@ function channelIdToTableRow(
   const weiTransferred = prettyPrintWei(utils.bigNumberify(channel.beneficiaryBalance));
 
   let connectionStatus;
-  if (wire && (channel.status === 'running' || channel.status === 'closing')) {
+  if (wire && channel.status === 'running') {
```

Comment:
The `if` statements can be reorganized to avoid having to re-check the existence of the wire.